### PR TITLE
Docker compose static testnet configuration

### PR DESCRIPTION
**Pull request by bacv**
Static docker compose configuration for deploying on a dedicated host.
Related issue: https://github.com/status-im/infra-misc/issues/189

### CODE REVIEW

File: compose.static.yml [hunk 1]
```diff
@@ -0,0 +1,137 @@
+services:
+
+  bootstrap:
```
Comment (youngjoon-lee) [inline compose.static.yml:3]: i think this also needs to depend on mixnodes.

**Review by youngjoon-lee — APPROVED** [review]
Looks good to me. Thank you!

File: compose.static.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
+services:
```
Comment (jakubgs) [inline compose.static.yml:1]: Compose files should specify scheme version, for example:
```suggestion
version: '3.7'
services:
```

File: compose.static.yml [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      - LIBP2P_NODE_MASK=${DOCKER_COMPOSE_LIBP2P_NODE_KEY_MASK:-2000000000000000000000000000000000000000000000000000000000000000}
+      - OVERLAY_NODES=${DOCKER_COMPOSE_OVERLAY_NODES:-1000000000000000000000000000000000000000000000000000000000000000}
+      - OVERLAY_SUPER_MAJORITY_THRESHOLD=${DOCKER_COMPOSE_SUPER_MAJORITY_THRESHOLD:-1}
+      - NET_INITIAL_PEERS=${DOCKER_COMPOSE_NET_INITIAL_PEERS:-/dns/bootstrap/tcp/3000}
+    entrypoint: /etc/nomos/scripts/run_nomos_node.sh
+
+  libp2p-node-3:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile
```
Comment (jakubgs) [inline compose.static.yml:76]: Why not just specify the path only?
```suggestion
    build: 'testnet'
```

**Review by jakubgs — APPROVED** [review]
I would recommend setting `container_name` for each of the containers, otherwise it the name will change depending on the name of the repo folder.

**Pull request merged**